src/sentry/static/sentry/app/views/admin/adminUserEdit.tsx

```diff
@@ -106,20 +111,22 @@ class RemoveUserModal extends React.Component {
           <Button priority="danger" onClick={this.onRemove}>
             {REMOVE_BUTTON_LABEL[deleteType]}
           </Button>
-          <Button onClick={this.props.closeModal}>{t('Nevermind')}</Button>
+          <Button onClick={this.props.closeModal}>{t('Cancel')}</Button>
         </ModalFooter>
       </React.Fragment>
     );
   }
 }
 
-class AdminUserEdit extends AsyncView {
+type Props = AsyncView['props'] & RouteComponentProps<{id: string}, {}>;
+
+class AdminUserEdit extends AsyncView<Props, AsyncView['state']> {
   get userEndpoint() {
     const {params} = this.props;
     return `/users/${params.id}/`;
   }
 
-  getEndpoints() {
+  getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
     return [['user', this.userEndpoint]];
```

Comment:
shouldn't we add a type for this state prop too?

Reply:
oh yes we should